src/test/suite/mdbExtensionController.test.ts

```diff
@@ -29,1891 +29,1744 @@ const testDatabaseURI = 'mongodb://localhost:27018';
+          );
+          mockVSCodeExecuteCommand = sinon.fake.resolves(undefined);
+          sinon.replace(
+            vscode.commands,
+            'executeCommand',
+            mockVSCodeExecuteCommand
+          );
+
+          void mdbTestExtension.testExtensionController.showOverviewPageIfRecentlyInstalled();
+        });
+
+        test('they are not shown the overview page', () => {
+          assert(!mockVSCodeExecuteCommand.called);
+        });
+      });
+    });
+  });
 
+  suite('with real text document', () => {
     beforeEach(() => {
+      sinon.stub(vscode.window, 'showInformationMessage');
+      sinon.stub(vscode.window, 'showErrorMessage');
+      const mockActiveConnectionId = sinon.fake.returns('tasty_sandwhich');
       sinon.replace(
-        mdbTestExtension.testExtensionController._storageController,
-        'get',
-        sinon.fake.returns(true)
+        mdbTestExtension.testExtensionController._connectionController,
+        'getActiveConnectionId',
+        mockActiveConnectionId
       );
+    });
 
-      mockVSCodeExecuteCommand = sinon.fake.resolves(undefined);
-      sinon.replace(
-        vscode.commands,
-        'executeCommand',
-        mockVSCodeExecuteCommand
+    afterEach(async () => {
+      await vscode.commands.executeCommand(
+        'workbench.action.closeActiveEditor'
+      );
+      sinon.restore();
+    });
+
+    test('mdb.addDatabase should create a MongoDB playground with create collection template', async () => {
+      const mockTreeItem = new ConnectionTreeItem(
+        'tasty_sandwhich',
+        vscode.TreeItemCollapsibleState.None,
+        false,
+        mdbTestExtension.testExtensionController._connectionController,
+        false,
+        {}
+      );
+      await vscode.commands.executeCommand('mdb.addDatabase', mockTreeItem);
+
+      const activeTextEditor =
+        mdbTestExtension.testExtensionController._playgroundController
+          ._activeTextEditor;
+      assert.strictEqual(activeTextEditor?.document.languageId, 'javascript');
+      assert(activeTextEditor?.document.uri.scheme === 'untitled');
+
+      const content = activeTextEditor.document.getText();
+      assert(content.includes('// Create a new database.'));
+      assert(content.includes('NEW_DATABASE_NAME'));
+      assert(content.includes('NEW_COLLECTION_NAME'));
+    });
+
+    test('mdb.addCollection should create a MongoDB playground with create collection template', async () => {
+      const mockTreeItem = new DatabaseTreeItem(
+        'iceCreamDB',
+        {},
+        false,
+        false,
+        {}
       );
+      await vscode.commands.executeCommand('mdb.addCollection', mockTreeItem);
+
+      const activeTextEditor =
+        mdbTestExtension.testExtensionController._playgroundController
+          ._activeTextEditor;
+      assert.strictEqual(activeTextEditor?.document.languageId, 'javascript');
+      assert(activeTextEditor?.document.uri.scheme === 'untitled');
```

Comment:
There’s still a few of these that could be fixed:

```suggestion
      assert.strictEqual(activeTextEditor?.document.uri.scheme, 'untitled');
```